src/components/InteractionLayer/index.js

```diff
@@ -129,7 +131,8 @@ class InteractionLayer extends React.Component {
       } else if (
         // ruler should follow points during live loading
         // except when the chart is dragging firing touchmove event
-        (((d3 || {}).event || {}).sourceEvent || {}).type === 'mousemove'
+        d3Type === 'mousemove' ||
+        d3Type === undefined
```

Comment:
Initially that was just `!== 'touchmove'`

Reply:
this is changed in #205 anyway